solc_json_parser/parser.py

```diff
@@ -377,7 +391,9 @@ def all_contract_names(self) -> List[str]:
     def all_abstract_contracts(self) -> List[ContractData]:
         return [contract for contract in self.all_contracts() if contract.abstract]
 
-    @cached_property
```

Comment:
any reason removing the `@cached_property`, I think we have to keep it to maintain the API compatibility